lib/protein/adapters/amqp_adapter/server.ex

```diff
@@ -91,8 +90,22 @@ defmodule Protein.AMQPAdapter.Server do
   defp init_conn_chan_queue(url, queue) do
     case Connection.open(url) do
       {:ok, conn} ->
-        {:ok, chan} = Channel.open(conn)
-        Queue.declare(chan, queue)
+        chan =
+          try do
+            {:ok, chan} = Channel.open(conn)
+            {:ok, _} = Queue.declare(chan, queue, durable: true)
+            Logger.info("Declared queue #{queue} as durable")
+            chan
+          catch
+            :exit, reason = {{:shutdown, {:server_initiated_close, 406, _message}}, _} ->
+              IO.inspect(reason)
```

Comment:
Some stale inspect.

Reply:
Nope, that's intented (for debugging/visibility reasons) but I can remove it if desired.

Reply:
If this is intended, let's keep it, but as `Logger.debug`.